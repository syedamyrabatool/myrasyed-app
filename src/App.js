import React from 'react';
import SC from './styles.js';
import UsersList from './Components/UsersList/index.js';
import {Provider} from 'react-redux';
import store from "./store/store.js";

function App() {
  return (
    <Provider store={store}>
       <SC.App>
        <UsersList enableSearch={true} />
      </SC.App>
    </Provider>
  );
}

export default App;
