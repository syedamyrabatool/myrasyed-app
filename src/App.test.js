import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  const component = renderer.create(<App />);
  const linkElement = component.toJSON()
  expect(linkElement).toMatchSnapshot();
});
