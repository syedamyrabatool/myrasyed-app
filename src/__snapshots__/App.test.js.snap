// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="sc-bdfBwQ evlUEU"
>
  <div
    className="sc-gsTCUz dngXxU"
  >
    <label
      className="sc-dlfnbm hOaEdj"
      htmlFor="search"
    >
      Search User
    </label>
    <input
      className="sc-hKgILt cTGSLc"
      data-testid="search"
      id="search"
      onChange={[Function]}
    />
  </div>
</div>
`;
